IFC4x3/Properties/g/GroundwaterAgressiveness_0eilySCHX2FwvuWJBgvmtA/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="GroundwaterAgressiveness_0eilySCHX2FwvuWJBgvmtA" Name="GroundwaterAgressiveness" UniqueId="28b2ff1c-3118-423f-ae78-8132eae70dca" PrimaryDataType="IfcLabel" />
```

Comment:
Is the rating a number?

Reply:
Not necessarily, country specific. I'd suggest label